indigo_app/templates/indigo_api/work_commencements.html

```diff
@@ -130,6 +130,13 @@ <h6>
               <div class="form-group provisions-commenced {% if commencement.all_provisions %}d-none{% endif %}">
                 <label>Provisions commenced:</label>
                 <small class="form-text text-muted">Use SHIFT and Ctrl (or ⌘) to select multiple items.</small>
+
+                <ul class="list-group" data-id="provisions">
+                  {% include "indigo_api/template_parts/provisions_tree_view.html" with provisions=commencement.possible_provisions %}
```

Comment:
what do you think of this implementation for nested lists. do you have a better way?